### ISSUE TITLE
Paket.Bootstraper.exe v5.243.0 returns 404 from GitHub

### ISSUE THREAD

**Issue by NSANUnik** [fsprojects/Paket#4276]
### Description

Paket.Bootstraper.exe from release 5.243.0 returns 404 from GitHub.

Paket.Bootstrapper.exe seems to be missing from release 9.0.0, meaning we can't update to newest version.

### Repro steps

Please provide the steps required to reproduce the problem

1. Run Paket.exe (Magic Mode - using bootstrapper v.5.243.0)

2. Error : (404) Not Found. (Github - cached (temporarily ignore updates))

**Comment by forki** [comment]
unfortunately .NET 9 doesn't support net461 packaging so that the bootstrapper needs a new design.
But dotnet tool is usually the way forward

**Comment by NightWatchman** [comment]
I'm not sure if this is the same thing as described by the original post, but when I install the latest tool version with `dotnet tool update paket` (9.0.0), I get 404 errors when I run `dotnet paket update`.

```
EXEC : The remote server returned an error : (404) Not Found. (Github - cached (temporarily ignore updates)) [D:\Developer\xxxx.xxxx.xxxx\Tests\Tests.csproj]
EXEC : The remote server returned an error : (404) Not Found. (Github - cached (temporarily ignore updates)) [D:\Developer\xxxx.xxxx.xxxx\xxxx.xxxx.xxxx.xxxx\xxxx.xxxx.xxxx.xxxx.csproj]
```

**Comment by GalaxyNetworks** [comment]
What confuses me the most is, that this change is a breaking change we haven't been informed of.

When I run an older paket.bootstrapper.exe in magic mode (paket.exe), I get the error 404 Not Found (Github - cached...) and it then tries to use the 9.0.0-alpha001 version to do the restore, even though it just failed with 404.

Does it successfully get that version from nuget.org instead, or what's going on?

I've looked at [https://fsprojects.github.io/Paket/bootstrapper.html](https://fsprojects.github.io/Paket/bootstrapper.html) but there's no mention of any breaking changes regarding the bootstrapper.

It seems wrong that our old pipelines should suddenly start failing, because an older bootstrapper cannot find a suitable version anymore.

We've fixed it by adding "version 8.0.3" to our paket.dependencies files, so it still works.
Some of our pipelines are old and won't be updated, so we'd either continue fixed on 8.0.3, or remove paket as a dependency.

I don't see any information about paket.bootstrapper.exe being deprecated or removed in 9.0.0, and even if it is, I would have assumed bootstrapper would be marked as deprecated and instead be locked to versions >= 8.0.3 so it wouldn't break existing pipelines and simply download 8.0.3 as the latest version while giving a warning that bootstrapper is deprecated.

This is a sudden breaking change - without any clear notice - so I assume this is a bug, right?
A bug that breaks bootstrapper if version isn't locked to 8.0.3 or lower.
